inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl

```diff
@@ -36,16 +36,25 @@
               </div>
               <div class="col-md-11">
                 <h4 class="custom-h">
+                  <b>
+                    {% if record['titles[0].title']|is_list() %}
+                      {% set title = record['titles[0].title[0]'] %}
+                    {% else %}
+                      {% set title = record['titles[0].title'] %}
+                    {% endif %}
+                    {% if not title %}
+                      {% set title = record['titles.title[0]'] %}
```

Comment:
As I remember in one record a title was not appearing. I checked and saw that titles.title was a list, so I added this is the case for this.